source/agora/common/API.d

```diff
@@ -151,7 +151,7 @@ public interface API
             GET /merkle_path
 
         Params:
-            block_height = the starting block height to begin retrieval from
+            block_height = block height with transaction hash
```

Comment:
```suggestion
            block_height = Height of the block that contains the transaction hash
```

Also you can change the commit title to "Fix documentation on API.getMerklePath"

Reply:
https://github.com/bpfkorea/agora/blob/v0.x.x/doc/Style.md#commits